app/pods/newsletter/nota-updates/route.js

```diff
@@ -0,0 +1,111 @@
+import Route from '@ember/routing/route';
+import moment from 'moment';
+import { hash } from 'rsvp';
+import { A } from '@ember/array';
+import CONFIG from 'fe-redpencil/utils/config';
+import getPaginationMetadata from 'fe-redpencil/utils/get-pagination-metadata';
+import {
+  task, timeout
+} from 'ember-concurrency';
+
+
+export default class NewsletterNotaUpdatesRoute extends Route {
+  queryParams = {
+    sort: {
+      refreshModel: true,
+    },
+  };
+
+  @(task(function *() {
+    while (true) {
+      yield timeout(0.1 * 60000);
+      this.refresh();
+    }
+  })) pollModel;
+
+  constructor() {
+    super(...arguments);
+    this.pollModel.perform();
+  }
+
+  async model(params) {
+    let notas = A([]);
+    const newsletterModel = this.modelFor('newsletter');
+    const meeting = newsletterModel.meeting;
+    const agenda = newsletterModel.agenda;
+    const agendaId = agenda.id;
+    const meetingId = meeting.id;
+    const agendaitemsForAgenda = await agenda.get('agendaitems');
+    const agendaitemsArray = agendaitemsForAgenda.toArray();
+    for (const agendaitem of agendaitemsArray) {
+      if (agendaitem.showAsRemark) {
```

Comment:
Voor externe lezer miscchien niet meteen obvious waarom dit hiet staat ... Een lijntje commentaar kan misschien helpen? :speech_balloon:

Reply:
done

Reply:
Ah, ik heb met niet duidelijk genoeg uitgedrukt ... ik had het eerder op waarom er op die `showAsRemark` vlag getest wordt ... iet in de stijl van `// Remarks by nature never contain documents of type "nota", it therefore doesn't make sense to process them any further`

Reply:
Daarna mag je van mij meteen mergen